content/en/docs/reference/glossary/static-pod.md

```diff
@@ -10,5 +10,4 @@ aka:
 tags:
 - fundamental
 ---
- A {{< glossary_tooltip text="pod" term_id="pod" >}} managed directly by the kubelet
- daemon on a specific node, without the API server observing it.
+ A Pod managed directly by the kubelet on a specific node, without the API server controlling it.
```

Comment:
hi @sftim. Did you get a clean, local build?
The preview looks like the other, correct previews in the past -- the `Mirror Pod` and `Static Pod` entries are in the glossary. Should the full_link field point to the new location for static-pod?
Question:
What happened to the glossary entries for `Mirror Pod` and `Static Pod` on the current kubernetes.io site?
What happened to the static-pod page when moved from tasks/administer-cluster to tasks/configure-pod-container/static-pod.md? It is odd that the preview is okay, but the site is not.

Reply:
Netlify built the PR and the preview looks OK, so that's enough to convince me. I didn't try this locally.

Good spot about the `full_link:`, I'll fix that.

Reply:
Would you have time to try and build locally -- I am really curious about the build failure :smiley:

Reply:
`make docker-serve` starts fine locally for this branch, and for _master_

Reply:
:+1:

Reply:
>  the mirror pod issue in the new static-pod file.
What issue is that @kbhawkey ? (I'm mostly concentrating elsewhere, sorry)

Reply:
Hi! I did a bit of testing. Lines 13 and 14 seem fine in my local hugo build (docker or hugo server). The date in static-pod and mirror-pod seem to cause an error.
See #15752.